lib/active_merchant/billing/gateways/iveri.rb

```diff
@@ -0,0 +1,240 @@
+require 'nokogiri'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class IveriGateway < Gateway
+      self.live_url = 'https://portal.nedsecure.co.za/iVeriWebService/Service.asmx'
```

Comment:
Guessing their test URL is the same as their live URL? I've seen this pattern in other adapters-

`self.live_url = self.test_url = 'https://example.com'`

Wondering if we should follow suit for the sake of consistency.